applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>Can we get a citation on the ABM tending to over predict compared to other methods? I found this from the <a href="http://www.bom.gov.au/info/thermal_stress/" target="_blank">ABM page</a>, but that's not exactly what you're saying here I don't think. </p><blockquote>The formula is likely to overestimate the&nbsp;<strong>WBGT</strong>&nbsp;in cloudy or windy conditions, or when the sun is low or below the horizon. Under clear full sun and low humidity conditions the approximation&nbsp;<em>underestimates</em>&nbsp;the&nbsp;<strong>WBGT</strong>&nbsp;slightly.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Yes that should be cited. I know I read it, but now I cannot find it when searching through all the readings in the Drive. I think for now I just remove that dubious statement.